### ISSUE TITLE
`ldbletanh=True` is untested

### ISSUE THREAD

**Issue by malmans2** [pyNEMO/pyDOMCFG#26]
I had a quick look at the codecov report: https://codecov.io/gh/pyNEMO/pyDOMCFG/src/34ca3ca8f750a9b1a94f32831c11274999790780/pydomcfg/domzgr/zco.py
I think we are missing a test for `ldbletanh = True`?

**Comment by oceandie** [comment]
Yes, we are, I couldn't find anything fast to compare with ... I will think about it

**Comment by malmans2** [comment]
Sounds good.
If the test will look very similar to orca2, this could be a good way to do it:
https://docs.pytest.org/en/6.2.x/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions

```python
@pytest.mark.parametrize("expected,ldbletanh,...", [(orca2_array, another_array), (True, False), ...])
def test_whatever(expected, ldbletanh, ...):
    ...
```
